.hound.yml

```diff
@@ -0,0 +1,3 @@
+eslint:
+ enabled: true
+ config_file: .eslintrc
```

Comment:
You have a eslintignore file but you did not reference it here for your hound.yml to use. That's a major reason for the 712 violations which hound is reporting.